src/main/webapp/app/entities/flair-bi/modal/modal-tabs/condition/condition.component.js

```diff
@@ -12,16 +12,16 @@
                 features: '=',
                 showAdd: '@',
                 showDelete: '@',
-                datasourceId:'='
+                datasourceId: '='
             }
         });
 
-    conditionComponent.$inject = ['$scope', 'CONDITION_TYPES', 'COMPARE_TYPES', '$rootScope', 'CryptoService','proxyGrpcService'];
+    conditionComponent.$inject = ['$scope', 'CONDITION_TYPES', 'COMPARE_TYPES', '$rootScope', 'CryptoService', 'proxyGrpcService', 'filterParametersService'];
 
-    function conditionComponent($scope, CONDITION_TYPES, COMPARE_TYPES, $rootScope, CryptoService,proxyGrpcService) {
+    function conditionComponent($scope, CONDITION_TYPES, COMPARE_TYPES, $rootScope, CryptoService, proxyGrpcService, filterParametersService) {
         var vm = this;
-        vm.load=load;
```

Comment:
if there is no change why this line is showing red...please change setting of ur editor...otherwise it is hard to indetify the changes